projects/js-packages/connection/components/connect-screen/layout/style.scss

```diff
@@ -86,8 +86,18 @@
 				padding-left: 30px;
 				margin-bottom: 9px;
 				color: var( --jp-black );
+
+				a {
+					color: #000;
+					text-decoration: underline;
+				}
 			}
 		}
+
+		a.jp-connection-screen-icon {
+			text-decoration: none;
+			padding-left: 5px;
+		}
```

Comment:
Reintroducing these lines, were lost in transition when we merged #21521.
They are responsible for styling the "See all Jetpack features" button and the "external" icon:

<img width="333" alt="Screen Shot 2021-11-04 at 10 44 06 AM" src="https://user-images.githubusercontent.com/1341249/140334914-6859cb74-d3b9-4687-bf3f-a23d8c2a1935.png">